kernel/device.c

```diff
@@ -84,17 +87,22 @@ const struct device *z_impl_device_get_binding(const char *name)
 	 * with CONFIG_* macros), only cheap pointer comparisons will be
 	 * performed. Reserve string comparisons for a fallback.
 	 */
+	LOG_DBG("device_get_binding(%s)", name);
```

Comment:
Yes but without that I don't have any indication that the function has started. It seems confusing to me.

Reply:
I don't think you need that info here. The function will either print that the device was found or not found in all cases, why would you need to know specifically that the function is started.

Reply:
If think you about the output when a device is not found, without the header it will be:

   - fred, ready=1
   - ernie, ready=1
   - mary, ready=1
   - lisa, ready=0
lisa not found

With this output, we get:

device_get_binding(lisa)
   - fred, ready=1
   - ernie, ready=1
   - mary, ready=1
   - lisa, ready=0
lisa not found

so I think it is clearer what is going on. What do you think?

Reply:
That is not the actual out you will see but something like this:
```
[00:00:00.110,000] <dbg> device.z_impl_device_get_binding: fred, ready=1
[00:00:00.110,000] <dbg> device.z_impl_device_get_binding: ernie, ready=1
[00:00:00.110,000] <dbg> device.z_impl_device_get_binding: mary, ready=1
[00:00:00.110,000] <dbg> device.z_impl_device_get_binding: lisa, ready=0
[00:00:00.110,000] <dbg> device.z_impl_device_get_binding: lisa not found
```
Printing extra
```
[00:00:00.110,000] <dbg> device.z_impl_device_get_binding: device_get_binding(lisa)
```
before this list does not give much more information to the user as you can already see the function name in the debug print headers, and the device name will be printed anyway (either "lisa found" or "lisa not found"). So imho the line could be omitted here.

Reply:
Well I don't agree. I think it is confusing to skip showing the start of the operation and require people to figure it out from implication.

Anyway I am new around here, and this is a really minor point, so let's go with what you say.